skbuild/cmaker.py

```diff
@@ -529,7 +529,7 @@ def _guess_python_library(python_version: str) -> str | None:
             candidate_suffixes = [""]
             candidate_implementations = ["python"]
             if hasattr(sys, "pypy_version_info"):
-                candidate_implementations = ["pypy-c", "pypy3-c", "pypy"]
+                candidate_implementations += ["pypy-c", "pypy3-c", "pypy"]
```

Comment:
This helps the tests pass when I add PyPy Windows, thanks! Quick question, though - Why isn't this prepended instead of appended? I'd assume `pypy` should be first in the search, before `python`?

Also, I think this check should be `sys.implementation.name == "pypy"`, but not a required change for this PR. :)

Reply:
```suggestion
                candidate_implementations[:0] = ["pypy-c", "pypy3-c", "pypy"]
```

Reply:
Yeah it *is* better on Linux and Mac with brew since they might end up with regular and pypy libs in the same folder. 👍